src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const successMessage = `<div class="message">Promise was resolved!</div>`;
+const errorMessage = `<div class="message error-message">Promise was rejected!</div>`;
+
+const successHandler = (message) => {
+  body.insertAdjacentHTML('beforeend', successMessage);
+};
```

Comment:
You are not using the `message` parameter in the `successHandler` function. It's better to use this parameter to make this function more flexible. For example, you can replace `successMessage` in the `insertAdjacentHTML` method with `message`.